ckanext/qa/model.py

```diff
@@ -75,11 +75,10 @@ def create(cls, resource_id):
         c.resource_id = resource_id
 
         # Find the package_id for the resource.
-        q = model.Session.query(model.Package.id)
+        q = model.Session.query(model.Resource.package_id)
```

Comment:
is this more efficient?

Reply:
It's more robust. In some cases attempting to load `model.Package` gives an error.